src/manifolds/ConnectionManifold.jl

```diff
@@ -335,6 +335,24 @@ function riemann_tensor(
 end
 @trait_function riemann_tensor(M::AbstractDecoratorManifold, p, B::AbstractBasis; kwargs...)
 
+@doc raw"""
```

Comment:
I thought we would have that already, but since we do not seems so – Nice to have it :) It would even be a candidate to be moved to `ManifoldsBase.jl` at some point.